### ISSUE TITLE
Add IRISPH5 to the list of known FDSN web service implementations

### ISSUE THREAD

**Issue by jrekoske-usgs** [obspy/obspy#2737]
IRISPH5 web services (https://service.iris.edu/ph5ws/) should be added to the list of known FDSN web service implementations. To access these services, one must currently supply the url, as well as the `service_mappings` argument when initializing the FDSN client.

```
from obspy.clients.fdsn import Client
irisph5_client = Client('http://service.iris.edu/ph5ws/', service_mappings={
     'dataselect': 'http://service.iris.edu/ph5ws/dataselect/1',
     'station': 'http://service.iris.edu/ph5ws/station/1'}
)
```

Obspy should be able to automatically discover this service by adding it to `obspy.clients.fdsn.header.URL_MAPPINGS` so that the client could be initialized by running

```irisph5_client = Client('IRISPH5')```

This would also make it nicer to use MassDownloader since it defaults to using the list of providers given in `URL_MAPPINGS`. To use IRISPH5 in the MassDownloader in combination with the other FDSN clients, one must first initalize the IRISPH5 client using the method above, and then create the MassDownloader using 

```
from obspy.clients.fdsn.mass_downloader import MassDownloader
from obspy.clients.fdsn.header import URL_MAPPINGS
mdl = MassDownloader(providers=[irisph5_client] + list(URL_MAPPINGS))
```

**Comment by megies** [comment]
Out of curiosity, what is this "IRISPH5"? Is this a separate project from the general archives?

We'll have to adapt our code a bit looks like, since we expect FDSN web services to adhere to the standard (i.e. base url, then `/fdsnws`, then e.g. `/dataselect`)..

**Comment by megies** [comment]
@jrekoske-usgs btw it seems the server doesn't deliver events in `text` as advertized by the landing pages:

![Screenshot from 2020-10-28 12-29-33](https://user-images.githubusercontent.com/1842780/97430586-6425ae80-1919-11eb-8be7-283653694154.png)


(I didn't check if there is a difference between what FDSNWS specs define for `text` and what IRIS delivers as `geocsv`)

**Comment by jrekoske-usgs** [comment]
@megies From what I understand, IRISPH5 is separate from the main IRIS-DMC. IRISPH5 is used mainly for active-source seismological datasets and uses the PH5 data format, as opposed to the main archive which uses the SEED format.  I'm not sure why the 'event' service doesn't return events in `text`, but I did test the `station` and `dataselect` services and they worked fine.

**Comment by ThomasLecocq** [comment]
It's mostly the array/nodal data no ? (massive H5 files) from e.g. FairField Nodal experiments?

**Comment by jrekoske-usgs** [comment]
@ThomasLecocq Right, it's used for the large nodal arrays. You can see the full list of experiments at http://ds.iris.edu/gmap/#network=*&datacenter=IRISPH5&planet=earth

**Comment by ThomasLecocq** [comment]
![image](https://user-images.githubusercontent.com/916937/97475055-1165ea00-194d-11eb-802e-14964d02512b.png)
53893 stations...

**Comment by aringler-usgs** [comment]
In case it is of any value, here is the PH5 repo: https://github.com/PIC-IRIS/PH5 @dsentinel knows a lot about this format.

**Comment by megies** [comment]
They still serve MiniSEED, though. But I guess they have it in PH5 on the server and it was cleaner to have a separate FDSNWS instance, which is understandable.

**Comment by megies** [comment]
fixed by #2739
